pkg/serverless/trace/trace.go

```diff
@@ -68,6 +68,9 @@ const dnsNonRoutableAddressURLPrefix = "0.0.0.0"
 // dnsLocalHostAddressURLPrefix is the first part of a URL from the localhost address for DNS traces
 const dnsLocalHostAddressURLPrefix = "127.0.0.1"
 
+// awsXrayDaemonAddressURLPrefix is the first part of a URL from the _AWS_XRAY_DAEMON_ADDRESS for DNS traces
+const awsXrayDaemonAddressURLPrefix = "169.254.79.129"
```

Comment:
I'm wondering if this ip is variable? Maybe we should instead parse the `_AWS_XRAY_DAEMON_ADDRESS` to determine this value?

Reply:
I feel okay about the assumption that this address will stay the same for aws lambda cases. but let me know if you feel strong about changing it. i'm okay either way.

Reply:
Sounds good, I was mostly asking as a question rather than as a statement of request. We can always do it this way, then if in the future we see the ip is changed, update the code then.

Reply:
definitely appreciate the thought. ❤️  i had the same idea and its "cost" (a tiny bit more logic) and "benefit"(a little bit more accurate) are both not that significant. That's why I felt I can go either way.